py/desisurvey/utils.py

```diff
@@ -343,7 +190,7 @@ def is_monsoon(night):
     """
     date = get_date(night)
     # Fetch our configuration.
-    config = desisurvey.config.Configuration()
+    config = Configuration()
```

Comment:
Style question: why did you want to change this from the explicit `desisurvey.config.Configuration` to `Configuration`?  It doesn't particularly matter here, but sometimes I prefer to keep the full path to make it clear where some function/object came from instead having to read back to the import statements to find out.  But you seem to prefer not including the full path.  Why?

Reply:
That's easy, because internal package imports should be relative, not absolute.  Thus at the top, there is a corresponding
```python
from .config import Configuration
```

Reply:
Following up on this: exactly why should internal package imports be relative, not absolute?  Is this driven by testing a checkout of a package that is already installed and not mixing the checkout with the installed version?

[PEP-8](https://www.python.org/dev/peps/pep-0008/) makes the opposite recommendation:

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved ... However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose

Either way, in future PRs please don't mix arguably style/taste changes with the core functionality that you are adding in the PR (in this case, moving freeze_iers).

Reply:
If we're quoting PEP-8: https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

Seriously though, I suspect that passage was written before the transition to Python 3 which significantly redefined the meaning of absolute and relative import.  The modern relative import notation makes it absolutely clear where the import is coming from.

If someone can code faster and more accurately while preferring a personal style, isn't that more important than slowing someone down by forcing them to adhere to a particular style?

Finally, if you want to enforce a particular style, we can simply add a mandatory `pycodestyle` or `flake8` test as part of the Travis tests.

Reply:
Throughout the DESI code we have used a mix of absolute and relative imports without requiring either style, leaving it up to the original authors to pick which method best suits their case.  In two cases recently you overrode the original author changing their absolute imports to relative imports in PRs that were otherwise unrelated, citing the reason "imports should be relative, not absolute".  That implied to me that you were advocating for enforcing a particular style, and I wanted to flesh out the reasons for that.  i.e. I'm fine with people using absolute or relative imports, but let's not post-facto override each other without clearly defining the reasons and what problem is being solved by changing what the original author did.